resolwe_bio/docker_images/legacy/packages-manual/r-packages.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash -e
+
+. /var/cache/build/packages-manual/common.sh
+
+# Install R packages.
+# XXX: This is unverifiable and thus may compromise the whole image.
+# XXX: Use notary (https://github.com/ropenscilabs/notary) when ready.
+sed -e 's/^#.*$//g' -e '/^$/d' /var/cache/build/packages-r.txt | \
+    Rscript --slave --no-save --no-restore-history \
+        -e "install.packages(readLines('stdin'), repos='https://cloud.r-project.org')"
+
+# XXX: This is unverifiable and in CLEAR TEXT and thus may compromise the whole image.
+# XXX: Use notary (https://github.com/ropenscilabs/notary) when ready.
+Rscript --slave --no-save --no-restore-history -e " \
+  package_list = c( \
+    'arrayQualityMetrics', \
```

Comment:
Should we make a list of R packages installed from bioconductor similar to `packages-r.txt`?